emails/tests/fixtures/domain_recipient_email_sns_body.json

```diff
@@ -0,0 +1 @@
kmFVwV86n29YbJQAPtPCoQfhhXfcICfswBMqmL\\r\\n\\t MZRgOIKjO4tYQ==\\r\\nTo: \\\"wildcard@subdomain.test.com\\\" <wildcard@subdomain.test.com>\\r\\nFrom: fxastage <fxastage@protonmail.com>\\r\\nReply-To: fxastage <fxastage@protonmail.com>\\r\\nSubject: localized email header + footer\\r\\nMessage-ID: <YQcRqVkgRtVPMmRcONFiK_YqawljgCRUfkTRngwYYI7lSsiE1IKax6R15p9gdSmKgSEXepeJTGhiwsTIJ-fWLA-Tc_lTb2BJdqAB0da4kx4=@protonmail.com>\\r\\nMIME-Version: 1.0\\r\\nContent-Type: multipart/alternative;\\r\\n boundary=\\\"b1_1tMoOzirX5rNUK0QIqMdSQB8hwcBeFGHtIfEtmvEG0k\\\"\\r\\nX-Spam-Status: No, score=-1.2 required=10.0 tests=ALL_TRUSTED,DKIM_SIGNED,\\r\\n\\tDKIM_VALID,DKIM_VALID_AU,DKIM_VALID_EF,FREEMAIL_FROM,HTML_MESSAGE\\r\\n\\tshortcircuit=no autolearn=disabled version=3.4.4\\r\\nX-Spam-Checker-Version: SpamAssassin 3.4.4 (2020-01-24) on\\r\\n\\tmailout.protonmail.ch\\r\\n\\r\\nThis is a multi-part message in MIME format.\\r\\n\\r\\n--b1_1tMoOzirX5rNUK0QIqMdSQB8hwcBeFGHtIfEtmvEG0k\\r\\nContent-Type: text/plain; charset=utf-8\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nc2hvdWxkIHdvcmsKClNlbnQgd2l0aCBbUHJvdG9uTWFpbF0oaHR0cHM6Ly9wcm90b25tYWlsLmNv\\r\\nbS8pIFNlY3VyZSBFbWFpbC4=\\r\\n\\r\\n--b1_1tMoOzirX5rNUK0QIqMdSQB8hwcBeFGHtIfEtmvEG0k\\r\\nContent-Type: text/html; charset=utf-8\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPGRpdiBzdHlsZT0iZm9udC1mYW1pbHk6IGFyaWFsOyBmb250LXNpemU6IDE0cHg7Ij48ZGl2IHN0\\r\\neWxlPSJmb250LWZhbWlseTogYXJpYWw7IGZvbnQtc2l6ZTogMTRweDsiPnNob3VsZCB3b3JrPGJy\\r\\nPjwvZGl2PjxkaXYgc3R5bGU9ImZvbnQtZmFtaWx5OiBhcmlhbDsgZm9udC1zaXplOiAxNHB4OyI+\\r\\nPGJyPjwvZGl2PjxkaXYgc3R5bGU9ImZvbnQtZmFtaWx5OiBhcmlhbDsgZm9udC1zaXplOiAxNHB4\\r\\nOyIgY2xhc3M9InByb3Rvbm1haWxfc2lnbmF0dXJlX2Jsb2NrIj48ZGl2IGNsYXNzPSJwcm90b25t\\r\\nYWlsX3NpZ25hdHVyZV9ibG9jay11c2VyIHByb3Rvbm1haWxfc2lnbmF0dXJlX2Jsb2NrLWVtcHR5\\r\\nIj48L2Rpdj48ZGl2IGNsYXNzPSJwcm90b25tYWlsX3NpZ25hdHVyZV9ibG9jay1wcm90b24iPlNl\\r\\nbnQgd2l0aCA8YSByZWw9Im5vb3BlbmVyIG5vcmVmZXJyZXIiIGhyZWY9Imh0dHBzOi8vcHJvdG9u\\r\\nbWFpbC5jb20vIiB0YXJnZXQ9Il9ibGFuayI+UHJvdG9uTWFpbDwvYT4gU2VjdXJlIEVtYWlsLjwv\\r\\nZGl2PjwvZGl2PjwvZGl2PjxkaXYgc3R5bGU9ImZvbnQtZmFtaWx5OiBhcmlhbDsgZm9udC1zaXpl\\r\\nOiAxNHB4OyI+PGJyPjwvZGl2Pg==\\r\\n\\r\\n\\r\\n--b1_1tMoOzirX5rNUK0QIqMdSQB8hwcBeFGHtIfEtmvEG0k--\\r\\n\\r\\n\"}", "Timestamp": "2022-01-13T21:02:46.911Z", "SignatureVersion": "1", "Signature": "BvlKJoLKcxcGsVloKnY7PKuqgY1EsSi1D5SVlHYtxRcsvuXVUDrdEvTlcWYNBiCpbaM/3kGYeXyb8uuWsdcNnT20zKLlpIP1dZhEV4o70aiM/35MxNfgAvA3NXkddRCQ2l+iplqtuRYdJ2XFkYJypECsoif9sRxB2jBW5bIz4vwjoeZdQCwMwBKQFsSGOr5Zc1pPE/5js9gZpBZlpsgRi2N4wj2sjBF0PyyO26UhGgWGwK1F7y1hpo90+PHRnD4W/XYc1flLu29hIaamgtXIbxol0xthMg0TtSfV9Q0MDJkpnsfOt/AsqvfZquIzGWyP2xpppuDLPf4MkftIbS4tpQ==", "SigningCertURL": "https://sns.us-east-1.amazonaws.com/SimpleNotificationService-7ff5318490ec183fbaddaa2a969abfda.pem", "UnsubscribeURL": "https://sns.us-east-1.amazonaws.com/?Action=Unsubscribe&SubscriptionArn=arn:aws:sns:us-east-1:168781634622:ses-inbound-grelay:f1e4420c-3b67-4cb2-b6db-4d22fd45916e"}
```

Comment:
Can you format this for humans?

Reply:
It doesn't do much, because the `Message` value itself is escaped json. ...

```
{
  "Type": "Notification",
  "MessageId": "36e7bb6d-268f-5b37-8c8f-e137b33903de",
  "TopicArn": "arn:aws:sns:us-east-1:168781634622:ses-inbound-grelay",
  "Subject": "Amazon SES Email Receipt Notification",
  "Message": "{\"notificationType\":\"Received\",\"mail\":{\"timestamp\":\"2022-01-13T21:02:45.441Z\",\"source\":\"fxastage@protonmail.com\",\"messageId\":\"s08vnjngg8o82cpus2084722eiate3c6ig885f01\",\"destination\":[\"wildcard@subdomain.test.com\"],\"headersTruncated\":false,\"headers\":[{\"name\":\"Return-Path\",\"value\":\"<fxastage@protonmail.com>\"},{\"name\":\"Received\",\"value\":\"from mail-4324.protonmail.ch (mail-4324.protonmail.ch [185.70.43.24]) by inbound-smtp.us-east-1.amazonaws.com with SMTP id s08vnjngg8o82cpus2084722eiate3c6ig885f01 for wildcard@subdomain.test.com; Thu, 13 Jan 2022 21:02:45 +0000 (UTC)\"},{\"name\":\"X-SES-Spam-Verdict\",\"value\":\"PASS\"},{\"name\":\"X-SES-Virus-Verdict\",\"value\":\"PASS\"},{\"name\":\"Received-SPF\",\"value\":\"pass (spfCheck: domain of protonmail.com designates 185.70.43.24 as permitted sender) client-ip=185.70.43.24; envelope-from=fxastage@protonmail.com; helo=mail-4324.protonmail.ch;\"},{\"name\":\"Authentication-Results\",\"value\":\"amazonses.com; spf=pass (spfCheck: domain of protonmail.com designates 185.70.43.24 as permitted sender) client-ip=185.70.43.24; envelope-from=fxastage@protonmail.com; helo=mail-4324.protonmail.ch; dkim=pass header.i=@protonmail.com; dmarc=pass header.from=protonmail.com;\"},{\"name\":\"X-SES-RECEIPT\",\"value\":\"AEFBQUFBQUFBQUFHNVEvWEd4Vnc5Vk1PbjZaNXM2aU40SG5IQ1ZGTjdKSlZnN1l0aWtIRjc1UnRsWnR0ZGMxcUpkQ2swNDgvRmRSTE5KRGxpZk80TkFNWjZyMVRCZ1IydzBEcGoyTTBGeDN1dnFVK1hwZ1d5ZVNMTlNrUnI0a1hmMXpFWXozVCtPcGRrdmxJOEdHQk1RVVFmZHAwaUxvdU5sK1ZGLzNiN0tHZjN6TFN2NzRFVGM0K054Q1VWRml5MUEyWUQ3bjdudmRGZkpLRGhxU3RrUXZJUU8veHpCSFJBNHFzdkdidWxYVVlUSHBad2hvSFBseWxsSEhFa2IzY09YbU05NjVQSCttWm9RS2MxSlBVQWxxaWtBTnFDaXBsTnBNWnpRQlRyN0hhMGZBOHpnT2hKOUE9PQ==\"},{\"name\":\"X-SES-DKIM-SIGNATURE\",\"value\":\"a=rsa-sha256; q=dns/txt; b=Dl/vQpjoU8O4WazhBVlg8hboteM60cDLYyNmm8YkzHJYBVsMpVY7vYHNvwf45hN7q8r+miiDnRaOcdsyE27RjCIZ3IB69+2Spit+ZDLLxpsX3HgfHhbW+0ULpnGJW/mzIcMZOD+lBsD+Uwg/wi08Qa0nqYh3zVcQq7tzQwFQxV0=; c=relaxed/simple; s=ug7nbtf4gccmlpwj322ax3p6ow6yfsug; d=amazonses.com; t=1642107766; v=1; bh=6aaytw5f3sf5609AJbSm6NdAf8MJLeZ89Zueo+Cyn5o=; h=From:To:Cc:Bcc:Subject:Date:Message-ID:MIME-Version:Content-Type:X-SES-RECEIPT;\"},{\"name\":\"Date\",\"value\":\"Thu, 13 Jan 2022 21:02:43 +0000\"},{\"name\":\"DKIM-Signature\",\"value\":\"v=1; a=rsa-sha256; c=relaxed/relaxed; d=protonmail.com; s=protonmail2; t=1642107763; bh=6aaytw5f3sf5609AJbSm6NdAf8MJLeZ89Zueo+Cyn5o=; h=Date:To:From:Reply-To:Subject:Message-ID:From:To:Cc; b=ZLcC5PqxvI8RZM1ayDZZqDnMIMj4PTc2X2Vz80p313tYJbatI7hRurerz6CBp5uI3IxQdX8SKA5eMcfsqSIV1MOxgbGUcV0q8n3V98qxdQhf+q4ylgllltc9TVRRA7Bs9K3J+bWMLUxWBxP/L4fFPDz1smoLLmEIPO1GzJ+SbfSw2fCb4PW0pSwqDtV6Cv6fagochgmwhfkV+4BQKbkAe+D19YU+7ZBw3Wt02PcOZEtlAdQZ3M3qLTCSX6dgZjr5j+2Q9PuE9wV5sCC/QiYjcTa65j9IRvmWNIKkmFVwV86n29YbJQAPtPCoQfhhXfcICfswBMqmLMZRgOIKjO4tYQ==\"},{\"name\":\"To\",\"value\":\"\\\"wildcard@subdomain.test.com\\\" <wildcard@subdomain.test.com>\"},{\"name\":\"From\",\"value\":\"fxastage <fxastage@protonmail.com>\"},{\"name\":\"Reply-To\",\"value\":\"fxastage <fxastage@protonmail.com>\"},{\"name\":\"Subject\",\"value\":\"localized email header + footer\"},{\"name\":\"Message-ID\",\"value\":\"<YQcRqVkgRtVPMmRcONFiK_YqawljgCRUfkTRngwYYI7lSsiE1IKax6R15p9gdSmKgSEXepeJTGhiwsTIJ-fWLA-Tc_lTb2BJdqAB0da4kx4=@protonmail.com>\"},{\"name\":\"MIME-Version\",\"value\":\"1.0\"},{\"name\":\"Content-Type\",\"value\":\"multipart/alternative; boundary=\\\"b1_1tMoOzirX5rNUK0QIqMdSQB8hwcBeFGHtIfEtmvEG0k\\\"\"},{\"name\":\"X-Spam-Status\",\"value\":\"No, score=-1.2 required=10.0 tests=ALL_TRUSTED,DKIM_SIGNED, DKIM_VALID,DKIM_VALID_AU,DKIM_VALID_EF,FREEMAIL_FROM,HTML_MESSAGE shortcircuit=no autolearn=disabled version=3.4.4\"},{\"name\":\"X-Spam-Checker-Version\",\"value\":\"SpamAssassin 3.4.4 (2020-01-24) on mailout.protonmail.ch\"}],\"commonHeaders\":{\"returnPath\":\"fxastage@protonmail.com\",\"from\":[\"fxastage <fxastage@protonmail.com>\"],\"replyTo\":[\"fxastage <fxastage@protonmail.com>\"],\"date\":\"Thu, 13 Jan 2022 21:02:43 +0000\",\"to\":[\"\\\"wildcard@subdomain.test.com\\\" <wildcard@subdomain.test.com>\"],\"messageId\":\"<YQcRqVkgRtVPMmRcONFiK_YqawljgCRUfkTRngwYYI7lSsiE1IKax6R15p9gdSmKgSEXepeJTGhiwsTIJ-fWLA-Tc_lTb2BJdqAB0da4kx4=@protonmail.com>\",\"subject\":\"localized email header + footer\"}},\"receipt\":{\"timestamp\":\"2022-01-13T21:02:45.441Z\",\"processingTimeMillis\":1432,\"recipients\":[\"wildcard@subdomain.test.com\"],\"spamVerdict\":{\"status\":\"PASS\"},\"virusVerdict\":{\"status\":\"PASS\"},\"spfVerdict\":{\"status\":\"PASS\"},\"dkimVerdict\":{\"status\":\"PASS\"},\"dmarcVerdict\":{\"status\":\"PASS\"},\"action\":{\"type\":\"SNS\",\"topicArn\":\"arn:aws:sns:us-east-1:168781634622:ses-inbound-grelay\",\"encoding\":\"UTF8\"}},\"content\":\"Return-Path: <fxastage@protonmail.com>\\r\\nReceived: from mail-4324.protonmail.ch (mail-4324.protonmail.ch [185.70.43.24])\\r\\n by inbound-smtp.us-east-1.amazonaws.com with SMTP id s08vnjngg8o82cpus2084722eiate3c6ig885f01\\r\\n for wildcard@subdomain.test.com;\\r\\n Thu, 13 Jan 2022 21:02:45 +0000 (UTC)\\r\\nX-SES-Spam-Verdict: PASS\\r\\nX-SES-Virus-Verdict: PASS\\r\\nReceived-SPF: pass (spfCheck: domain of protonmail.com designates 185.70.43.24 as permitted sender) client-ip=185.70.43.24; envelope-from=fxastage@protonmail.com; helo=mail-4324.protonmail.ch;\\r\\nAuthentication-Results: amazonses.com;\\r\\n spf=pass (spfCheck: domain of protonmail.com designates 185.70.43.24 as permitted sender) client-ip=185.70.43.24; envelope-from=fxastage@protonmail.com; helo=mail-4324.protonmail.ch;\\r\\n dkim=pass header.i=@protonmail.com;\\r\\n dmarc=pass header.from=protonmail.com;\\r\\nX-SES-RECEIPT: AEFBQUFBQUFBQUFHNVEvWEd4Vnc5Vk1PbjZaNXM2aU40SG5IQ1ZGTjdKSlZnN1l0aWtIRjc1UnRsWnR0ZGMxcUpkQ2swNDgvRmRSTE5KRGxpZk80TkFNWjZyMVRCZ1IydzBEcGoyTTBGeDN1dnFVK1hwZ1d5ZVNMTlNrUnI0a1hmMXpFWXozVCtPcGRrdmxJOEdHQk1RVVFmZHAwaUxvdU5sK1ZGLzNiN0tHZjN6TFN2NzRFVGM0K054Q1VWRml5MUEyWUQ3bjdudmRGZkpLRGhxU3RrUXZJUU8veHpCSFJBNHFzdkdidWxYVVlUSHBad2hvSFBseWxsSEhFa2IzY09YbU05NjVQSCttWm9RS2MxSlBVQWxxaWtBTnFDaXBsTnBNWnpRQlRyN0hhMGZBOHpnT2hKOUE9PQ==\\r\\nX-SES-DKIM-SIGNATURE: a=rsa-sha256; q=dns/txt; b=Dl/vQpjoU8O4WazhBVlg8hboteM60cDLYyNmm8YkzHJYBVsMpVY7vYHNvwf45hN7q8r+miiDnRaOcdsyE27RjCIZ3IB69+2Spit+ZDLLxpsX3HgfHhbW+0ULpnGJW/mzIcMZOD+lBsD+Uwg/wi08Qa0nqYh3zVcQq7tzQwFQxV0=; c=relaxed/simple; s=ug7nbtf4gccmlpwj322ax3p6ow6yfsug; d=amazonses.com; t=1642107766; v=1; bh=6aaytw5f3sf5609AJbSm6NdAf8MJLeZ89Zueo+Cyn5o=; h=From:To:Cc:Bcc:Subject:Date:Message-ID:MIME-Version:Content-Type:X-SES-RECEIPT;\\r\\nDate: Thu, 13 Jan 2022 21:02:43 +0000\\r\\nDKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=protonmail.com;\\r\\n\\ts=protonmail2; t=1642107763;\\r\\n\\tbh=6aaytw5f3sf5609AJbSm6NdAf8MJLeZ89Zueo+Cyn5o=;\\r\\n\\th=Date:To:From:Reply-To:Subject:Message-ID:From:To:Cc;\\r\\n\\tb=ZLcC5PqxvI8RZM1ayDZZqDnMIMj4PTc2X2Vz80p313tYJbatI7hRurerz6CBp5uI3\\r\\n\\t IxQdX8SKA5eMcfsqSIV1MOxgbGUcV0q8n3V98qxdQhf+q4ylgllltc9TVRRA7Bs9K3\\r\\n\\t J+bWMLUxWBxP/L4fFPDz1smoLLmEIPO1GzJ+SbfSw2fCb4PW0pSwqDtV6Cv6fagoch\\r\\n\\t gmwhfkV+4BQKbkAe+D19YU+7ZBw3Wt02PcOZEtlAdQZ3M3qLTCSX6dgZjr5j+2Q9Pu\\r\\n\\t E9wV5sCC/QiYjcTa65j9IRvmWNIKkmFVwV86n29YbJQAPtPCoQfhhXfcICfswBMqmL\\r\\n\\t MZRgOIKjO4tYQ==\\r\\nTo: \\\"wildcard@subdomain.test.com\\\" <wildcard@subdomain.test.com>\\r\\nFrom: fxastage <fxastage@protonmail.com>\\r\\nReply-To: fxastage <fxastage@protonmail.com>\\r\\nSubject: localized email header + footer\\r\\nMessage-ID: <YQcRqVkgRtVPMmRcONFiK_YqawljgCRUfkTRngwYYI7lSsiE1IKax6R15p9gdSmKgSEXepeJTGhiwsTIJ-fWLA-Tc_lTb2BJdqAB0da4kx4=@protonmail.com>\\r\\nMIME-Version: 1.0\\r\\nContent-Type: multipart/alternative;\\r\\n boundary=\\\"b1_1tMoOzirX5rNUK0QIqMdSQB8hwcBeFGHtIfEtmvEG0k\\\"\\r\\nX-Spam-Status: No, score=-1.2 required=10.0 tests=ALL_TRUSTED,DKIM_SIGNED,\\r\\n\\tDKIM_VALID,DKIM_VALID_AU,DKIM_VALID_EF,FREEMAIL_FROM,HTML_MESSAGE\\r\\n\\tshortcircuit=no autolearn=disabled version=3.4.4\\r\\nX-Spam-Checker-Version: SpamAssassin 3.4.4 (2020-01-24) on\\r\\n\\tmailout.protonmail.ch\\r\\n\\r\\nThis is a multi-part message in MIME format.\\r\\n\\r\\n--b1_1tMoOzirX5rNUK0QIqMdSQB8hwcBeFGHtIfEtmvEG0k\\r\\nContent-Type: text/plain; charset=utf-8\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nc2hvdWxkIHdvcmsKClNlbnQgd2l0aCBbUHJvdG9uTWFpbF0oaHR0cHM6Ly9wcm90b25tYWlsLmNv\\r\\nbS8pIFNlY3VyZSBFbWFpbC4=\\r\\n\\r\\n--b1_1tMoOzirX5rNUK0QIqMdSQB8hwcBeFGHtIfEtmvEG0k\\r\\nContent-Type: text/html; charset=utf-8\\r\\nContent-Transfer-Encoding: base64\\r\\n\\r\\nPGRpdiBzdHlsZT0iZm9udC1mYW1pbHk6IGFyaWFsOyBmb250LXNpemU6IDE0cHg7Ij48ZGl2IHN0\\r\\neWxlPSJmb250LWZhbWlseTogYXJpYWw7IGZvbnQtc2l6ZTogMTRweDsiPnNob3VsZCB3b3JrPGJy\\r\\nPjwvZGl2PjxkaXYgc3R5bGU9ImZvbnQtZmFtaWx5OiBhcmlhbDsgZm9udC1zaXplOiAxNHB4OyI+\\r\\nPGJyPjwvZGl2PjxkaXYgc3R5bGU9ImZvbnQtZmFtaWx5OiBhcmlhbDsgZm9udC1zaXplOiAxNHB4\\r\\nOyIgY2xhc3M9InByb3Rvbm1haWxfc2lnbmF0dXJlX2Jsb2NrIj48ZGl2IGNsYXNzPSJwcm90b25t\\r\\nYWlsX3NpZ25hdHVyZV9ibG9jay11c2VyIHByb3Rvbm1haWxfc2lnbmF0dXJlX2Jsb2NrLWVtcHR5\\r\\nIj48L2Rpdj48ZGl2IGNsYXNzPSJwcm90b25tYWlsX3NpZ25hdHVyZV9ibG9jay1wcm90b24iPlNl\\r\\nbnQgd2l0aCA8YSByZWw9Im5vb3BlbmVyIG5vcmVmZXJyZXIiIGhyZWY9Imh0dHBzOi8vcHJvdG9u\\r\\nbWFpbC5jb20vIiB0YXJnZXQ9Il9ibGFuayI+UHJvdG9uTWFpbDwvYT4gU2VjdXJlIEVtYWlsLjwv\\r\\nZGl2PjwvZGl2PjwvZGl2PjxkaXYgc3R5bGU9ImZvbnQtZmFtaWx5OiBhcmlhbDsgZm9udC1zaXpl\\r\\nOiAxNHB4OyI+PGJyPjwvZGl2Pg==\\r\\n\\r\\n\\r\\n--b1_1tMoOzirX5rNUK0QIqMdSQB8hwcBeFGHtIfEtmvEG0k--\\r\\n\\r\\n\"}",
  "Timestamp": "2022-01-13T21:02:46.911Z",
  "SignatureVersion": "1",
  "Signature": "BvlKJoLKcxcGsVloKnY7PKuqgY1EsSi1D5SVlHYtxRcsvuXVUDrdEvTlcWYNBiCpbaM/3kGYeXyb8uuWsdcNnT20zKLlpIP1dZhEV4o70aiM/35MxNfgAvA3NXkddRCQ2l+iplqtuRYdJ2XFkYJypECsoif9sRxB2jBW5bIz4vwjoeZdQCwMwBKQFsSGOr5Zc1pPE/5js9gZpBZlpsgRi2N4wj2sjBF0PyyO26UhGgWGwK1F7y1hpo90+PHRnD4W/XYc1flLu29hIaamgtXIbxol0xthMg0TtSfV9Q0MDJkpnsfOt/AsqvfZquIzGWyP2xpppuDLPf4MkftIbS4tpQ==",
  "SigningCertURL": "https://sns.us-east-1.amazonaws.com/SimpleNotificationService-7ff5318490ec183fbaddaa2a969abfda.pem",
  "UnsubscribeURL": "https://sns.us-east-1.amazonaws.com/?Action=Unsubscribe&SubscriptionArn=arn:aws:sns:us-east-1:168781634622:ses-inbound-grelay:f1e4420c-3b67-4cb2-b6db-4d22fd45916e"
}
```